fotoapparat/src/main/java/io/fotoapparat/hardware/v1/Camera1.java

```diff
@@ -176,6 +179,8 @@ public void updateParameters(Parameters parameters) {
         );
 
         camera.setParameters(cameraParameters);
+
+        previewScaleType = parameters.getValue(Type.PREVIEW_SCALE_TYPE);
```

Comment:
We are passing parameter only to return it back again. I think it could be a code smell which is caused by the fact that scale type is not a property of the camera - it is a property of the view.